### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by Nieminik**
[coverage.txt](https://github.com/everycache-group/everycache-app/files/7762977/coverage.txt)

### CODE REVIEW

**Comment by Nieminik** [comment]
@Yamsha75 We basically have full coverage as of now. Please review the changes

**Comment by Nieminik** [comment]
Fixed the tests.
97% coverage as of now.
@Yamsha75 Let's get this one over with.

File: api/everycache_api/tests/factories/cache_comment_factory.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (Yamsha75) [inline api/everycache_api/tests/factories/cache_comment_factory.py:1]: Please remove this emty line

File: api/everycache_api/tests/factories/cache_visit_factory.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (Yamsha75) [inline api/everycache_api/tests/factories/cache_visit_factory.py:1]: Please remove this empty line

File: api/everycache_api/tests/helpers.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (Yamsha75) [inline api/everycache_api/tests/helpers.py:1]: Please remove this empty line

**Review by Yamsha75 — CHANGES_REQUESTED** [review]
Apart from a few empty lines at the beginning of certain files the only issue I found was not updating Pipfile.lock. Please do that and we'll be ready for merge

**Review by Yamsha75 — APPROVED** [review]

**Pull request merged**